workspaces/rollbar/plugins/rollbar/src/hooks/useProject.ts

```diff
@@ -33,5 +36,6 @@ export function useProjectSlugFromEntity(entity: Entity) {
       organization ??
       configApi.getOptionalString('rollbar.organization') ??
       configApi.getString('organization.name'),
+    environment: entity?.metadata.annotations?.[ROLLBAR_ENVIRONMENT_ANNOTATION],
```

Comment:
Should we be setting the default here?

Reply:
It's actually already defined in the API client: https://github.com/backstage/community-plugins/blob/0b36d1a9a00c3f2a0c071f6d4fa6b1bd68edde89/workspaces/rollbar/plugins/rollbar/src/api/RollbarClient.ts#L49

Reply:
Ah, I see, wasn't in any of the changes so guess that's why I missed it. Makes sense now 👍